### PR TITLE
Base Stats Tab Styling

### LINKED ISSUE #67: Finalize Modal BaseStats Tab Styling
**Is your feature request related to a problem? Please describe.**
Finalize styling for base stats modal tab

**Describe the solution you'd like**
Contain bar graph within modal area and polish overall styling

### PR DESCRIPTION
**Pull request by Gjhnsn**
## Changes
1. Added container for graph
2. Adjusted bar thickness

## Purpose
_Describe the problem or feature in addition to a link to the issues._
Graph needed a container to keep within modal dimensions and to center it within the modal.

Closes #67

### CODE REVIEW

File: src/components/BaseStats.js [hunk 1]
```diff
@@ -11,10 +11,7 @@ const BaseStats = ({ stats, types }) => {
 
   return (
     <>
-      <div className="header">
-        <div className="links"></div>
-      </div>
-      <div>
+      <div className="graph-container">
```
Comment (BlackCubes) [inline src/components/BaseStats.js:14]: Interesting name

**Review by Lap343 — APPROVED** [review]
You weren't joking. Code looks good

**Review by BlackCubes — APPROVED** [review]
Looks gewd!

**Review by JorgeLVilla — APPROVED** [review]
Nice job!

**Review by dannysanchez559 — APPROVED** [review]

**Pull request merged**